src/components/SelectionList/index.tsx

```diff
@@ -42,6 +43,8 @@ function SelectionList<TItem extends ListItem>({onScroll, ...props}: BaseSelecti
             {...props}
             ref={ref}
             onScroll={onScroll ?? defaultOnScroll}
+            // Ignore the focus if it's caused by a touch event on a mobile chrome
+            shouldIgnoreFocus={Browser.isMobileChrome() && isScreenTouched}
```

Comment:
So, I found that the long press callback is called later than the onFocus, so I changed the solution to ignore the focus whenever the touch event is detected. Luckily we already detect the touch event in this file, so I just need to pass down the shouldIgnoreFocus prop and ignore the focus if it's true.

Reply:
It might be more helpful later if the comment clearly mentions the quirk of Android Chrome triggering the onFocus event on long press.

Reply:
Updated